packages/sui-studio/bin/sui-studio-link.js

```diff
@@ -6,6 +6,8 @@ const fs = require('fs-extra')
 const program = require('commander')
 const BASE_DIR = process.cwd()
 
+console.log(colors.yellow('sui-studio link is deprecated. Use "sui-studio link-all" instead.'))
```

Comment:
Makes sense to deprecate link?

link-all will allow us to link all packages in the same studio, but it wont allow to link packages to other studios, while link does